### PR TITLE
Added Abeg Logo and Details 

### PR DESCRIPTION
**Pull request by ichtrojan**
# Pull Request Contains
* Abeg Logo (PNG & SVG)
* Abeg Details (logo.json)
* Fixes to build errors on macOS Catalina

### CODE REVIEW

**Comment by ichtrojan** [comment]
> Nice @ichtrojan
> Please ensure your logos have been properly optimised with tools like: [SVGOMG](https://jakearchibald.github.io/svgomg/) & [Squoosh](https://squoosh.app/) (OptiPNG works well).
> 
> You can check [the contribution document](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) for more info

I have optimised the Assets

File: package.json [hunk 1]
```diff
@@ -21,20 +21,20 @@
...
   "devDependencies": {
-    "node-sass": "^4.11.0",
     "eslint": "^6.1.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.3.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-prettier": "^3.1.0",
     "husky": "^3.0.5",
     "lint-staged": "^9.2.5",
+    "node-sass": "^4.13.1",
```
Comment (Odusanya) [inline package.json]: Hey @ichtrojan just curious, is there any specific reason why you updated the node-sass version? Did you run into any form of error with the previous version?

**Comment by ichtrojan** [comment]
Hi @demilad, any Idea when this will be merged?

File: package.json (hunk 1 above)
Comment (demilad) [reply package.json]: @ichtrojan

**Comment by Odusanya** [comment]
> Hi @demilad, any Idea when this will be merged?

Hey Micheal, sorry for the hold-up, as soon as we're clear on this. Your PR will get merged -> https://github.com/PaystackHQ/nigerialogos/pull/130#discussion_r379381407

Ideally, PR's shouldn't come with an edited package.json or package-lock.json file.

**Comment by ichtrojan** [comment]
> > Hi @demilad, any Idea when this will be merged?
> 
> Hey Micheal, sorry for the hold-up, as soon as we're clear on this. Your PR will get merged -> [#130 (comment)](https://github.com/PaystackHQ/nigerialogos/pull/130#discussion_r379381407)
> 
> Ideally, PR's shouldn't come with an edited package.json or package-lock.json file.

Yeah, I updated the lock file to fix build issues on MacOS Catalina. Initially the error I got was something related to architecture not supported; the additions to the lock file fixed the issue on my machine and it will also prevent the issue from happening to other developers in the future.

File: package.json (hunk 1 above)
Comment (ichtrojan) [reply package.json]: Yes, it failed to build on macOS Catalina.

Thanks for the merge.

**Pull request merged**